addon/serializer-registry.js

```diff
@@ -96,21 +96,22 @@ export default class SerializerRegistry {
     // Traverse this model's relationships
     let serializer = this._serializerFor(model);
     serializer.relationships
-      .map(key => model[key])
-      .forEach(relationship => {
+      .filter(key => model[key])
+      .forEach(relationshipKey => {
+        let relationship = model[relationshipKey];
```

Comment:
changed so I can access the relationshipKey and pass it through to the functions below.